setup.py

```diff
@@ -11,13 +11,36 @@
 from rpm_py_installer.version import VERSION
 
 
+def run_cmd(cmd):
+    """Run a command."""
+    exit_status = 0
+    message = ''
+
+    proc = subprocess.Popen(
+        cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    proc.wait()
+    exit_status = proc.returncode
+    stdout = proc.stdout.read().decode()
+    stderr = proc.stderr.read().decode()
+    message = '''
+Command: {0}
+Return Code: [{1}]
+Stdout: [{2}]
+Stderr: [{3}]
+'''.format(cmd, exit_status, stdout, stderr)
```

Comment:
Tip: you can use [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) for nicer formatting.

Reply:
Thanks for telling it. I didn't know that the function. I will fix it on your way. I could see the function is available in Python 3.4 and 2.7 too.
https://docs.python.org/3.4/library/textwrap.html#textwrap.dedent
https://docs.python.org/2/library/textwrap.html#textwrap.dedent

Reply:
I tried the code below on my local.

```
...
import textwrap
...
    message = '''\
    Command: {0}
    Return Code: [{1}]
    Stdout: [{2}]
    Stderr: [{3}]
    '''.format(cmd, exit_status, stdout, stderr)
    message = textwrap.dedent(message)
````

But it seems the message still has indent. Maybe this improvement is for another PR.

```
(venv) $ pip install --use-pep517 .
...
      Exception:     Command: /home/jaruga/var/git/rpm-py-installer/venv/bin/python3 install.py
          Return Code: [1]
          Stdout: [[INFO] Installing...
      [INFO] Created working directory '/tmp/tmp2ms3e_1d-rpm-py-installer'
      [INFO] Downloading archive. 'https://github.com/rpm-software-management/rpm/archive/rpm-4.18.0-release.tar.gz'.
      ]
          Stderr: [Traceback (most recent call last):
        File "/home/jaruga/var/git/rpm-py-installer/install.py", line 2224, in <module>
          main()
        File "/home/jaruga/var/git/rpm-py-installer/install.py", line 2219, in main
          app.run()
        File "/home/jaruga/var/git/rpm-py-installer/install.py", line 46, in run
          raise InstallError(message)
      InstallError: RPM Python binding failed to install with unknown reason.
      ]
````

Reply:
I think you should dedent only the string literal, not the formatted message:

```python
    message = textwrap.dedent('''\
    Command: {0}
    Return Code: [{1}]
    Stdout: [{2}]
    Stderr: [{3}]
    ''').format(cmd, exit_status, stdout, stderr)
```